app/src/components/pages/divisionPage/divisionPage.tsx

```diff
@@ -1,25 +1,62 @@
 "use client";
-import { useParams } from "next/navigation";
 import {
   Box,
   Heading,
   HorizontalRule,
   Link,
+  Pagination,
 } from "@nypl/design-system-react-components";
+import {
+  useParams,
+  useSearchParams,
+  usePathname,
+  useRouter,
+} from "next/navigation";
+import React, { useEffect, useState, useRef } from "react";
 import PageLayout from "../../pageLayout/pageLayout";
 import { headerBreakpoints } from "../../../utils/breakpoints";
-import { slugToString } from "../../../utils/utils";
-import React, { useEffect, useState } from "react";
-import { ItemLane } from "../../lanes/itemLane/itemLane";
+// import { ItemLane } from "../../lanes/itemLane/itemLane";
 import { CollectionsGrid } from "../../grids/collectionsGrid";
-import CollectionLanesLoading from "../../lanes/collectionLanes/collectionLanesLoading";
+// import CollectionLanesLoading from "../../lanes/collectionLanes/collectionLanesLoading";
+import { slugToString, totalNumPages } from "../../../utils/utils";
+import useBreakpoints from "../../../hooks/useBreakpoints";
+// import { slugToString } from "../../../utils/utils";
+// import React, { useEffect, useState } from "react";
+// import { CollectionsGrid } from "../../grids/collectionsGrid";
```

Comment:
Same here, remove commented out code.

Reply:
yeah sorry I forgot to save these files rip